CHANGELOG.md

```diff
@@ -99,10 +99,12 @@
 - Add privacy policy to site
 - Empty optional dates for `actual start date` and `actual end date` are not included on the activity XML
 - Reporting org in the IATI XML is always BEIS for funds, programmes and projects created by governmental organisations, and the activity's organisation if it is a non-governmental organisation
-- User actions are tracked on Activities, Budgets, Transactions and Users.
 
 ## [unreleased]
 
+- User actions are tracked on Activities, Budgets, Transactions and Users.
```

Comment:
I don't know how this one ended up in `released` after the rebase with develop in #434, but it has definitely not been released!